portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -5,9 +5,11 @@
   private String comment;
   private String posterName;
 
-  public Comment(String comment, String posterName) {
+  public Comment(String posterName, String comment) {
```

Comment:
If you decide to increase the number of parameters for the Comment class consider looking into a Builder

Reply:
I'll definitely consider it. I was considering using a Comment class builder when I initially made the class. If I end up adding enough fields, I may implement a builder.